contracts/test/VotableSchemeMock.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity ^0.5.17;
+
+import "../votingMachines/GenesisProtocol.sol";
+import "../votingMachines/VotingMachineCallbacksInterface.sol";
+import "../votingMachines/ProposalExecuteInterface.sol";
+import "../Reputation.sol";
+import "./Debug.sol";
+
+
+contract VotableSchemeMock is Debug, VotingMachineCallbacksInterface, ProposalExecuteInterface, GenesisProtocol {
```

Comment:
does VotingMachineCallbacksInterface and  ProposalExecuteInterface are required ?

Reply:
Well yes, because its a scheme...